src/main/java/Duke.java

```diff
@@ -1,10 +1,60 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import javafx.application.Application;
+import javafx.stage.Stage;
+
+/**
+ * Main class duke to start the programme.
+ */
+public class Duke extends Application {
```

Comment:
Perhaps could try removing the start function and the inheritance from Application as all is handled in MainWindow, to make the code neater.